packages/client/hmi-client/src/App.vue

```diff
@@ -25,6 +34,29 @@ const disableOverlay = () => {
 };
 
 const dataExplorerActivated = ref(false);
+
+/**
+ * Project
+ *
+ * As we use only one Project per application instance.
+ * It is loaded at the root and passed to all views as prop.
+ */
+const project = ref<Project | null>();
```

Comment:
Genuinely curious, don't we need to explicitly initialize this to `null`?

```suggestion
const project = ref<Project | null>(null);
```

I'd assume that otherwise it's initialized to `undefined` which would unexpectedly fail any checks for `project.value === null`.

But I also assume TypeScript would yell at you if it was initialized to `undefined`...? Curious about the actual behaviour.

Reply:
I can't find documentation about it, so I added the `null`

Reply:
Yeah looks like it gets initialized to `undefined` and the type gets (silently) extended to `Project | null | undefined`.

Adding `null` is the way to go. 👍